remo/domain/annotation.py

```diff
@@ -55,6 +63,7 @@ def bbox(self, values: List[int]):
 
         self.object = Annotation.Bbox(*values)
         self.coordinates = values
+        self.type = 'Bounding Box'
```

Comment:
I think `type` relates to Annotation.Bbox class, and should be moved there

similar for Annotation.Segment

Reply:
@vmanilo it's just that we don't have a sub class for image classification at the moment. Also not clear how to handle tags from code yet - I would leave this for a later smaller refactoring, do you agree?

Reply:
ok